setup.py

```diff
@@ -84,93 +84,92 @@
         'manageiq.appliance_collections':
         [
             'actions = cfme.control.explorer.actions:ActionCollection',
-            'alerts = cfme.control.explorer.alerts:AlertCollection',
             'alert_profiles = cfme.control.explorer.alert_profiles:AlertProfileCollection',
-            'conditions = cfme.control.explorer.conditions:ConditionCollection',
-            ('diagnostic_workers = '
-                'cfme.configure.configuration.diagnostics_settings:DiagnosticWorkersCollection'),
-            'policies = cfme.control.explorer.policies:PolicyCollection',
-            'policy_profiles = cfme.control.explorer.policy_profiles:PolicyProfileCollection',
+            'alerts = cfme.control.explorer.alerts:AlertCollection',
```

Comment:
**QUESTION:**  What are you doing in here?   Alphabetizing?

Reply:
Correct, sorting the collections.